backend/models/GpsLocation.js

```
@@ -0,0 +1,200 @@
+    return result.rows.map(row => new GpsLocation(row));
+  }
+
+  // Obtenir l'historique des positions pour une date spécifique
+  static async getLocationsByDate(livreur_id, date) {
+    const startOfDay = new Date(date);
+    startOfDay.setHours(0, 0, 0, 0);
+    
+    const endOfDay = new Date(date);
+    endOfDay.setHours(23, 59, 59, 999);
+    
+    return this.getLocationHistory(livreur_id, startOfDay, endOfDay);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)